package.json

```diff
@@ -5,13 +5,8 @@
   "main": "index.js",
   "type": "module",
   "scripts": {
-    "start": "nodemon index.js",
-    "task1": "nodemon module_3/task1.js",
-    "task2": "nodemon module_3/task2.js",
-    "task3": "nodemon module_3/task3.js",
-    "test": "jest src/test/unit src/test/integration",
-    "test:coverage": "jest src/test/unit src/test/integration --coverage",
-    "test:e2e": "jest src/test/e2e"
+    "start": "./node_modules/.bin/pm2 start index.js --name ngmp-network-app",
+    "stop": "./node_modules/.bin/pm2 stop ngmp-network-app"
```

Comment:
I know this hint was provided by the task, however, the binaries can be used without specifying the full path, the advantage of using package.json is it is able to read binaries and avoids you to specify a full path, so you could use like:

`pm2 start index.js --name ngmp-network-app`

Not changes requested